wallet/src/main/scala/org/bitcoins/wallet/api/WalletApi.scala

```diff
@@ -51,6 +66,116 @@ sealed trait WalletApi {
   def chainParams: ChainParams = walletConfig.chain
 
   def networkParameters: NetworkParameters = walletConfig.network
+
+  case class SerializedTransactionInput(
```

Comment:
All the `decodeRawTransaction` stuff should be moved somewhere else, the `WalleApi`trait most just holds method signatures, not actual definitions